### DISCUSSION TITLE
How to convert RAR5 archives to ZIP ?

### DISCUSSION THREAD

**Discussion by gotson** [Flexget/Flexget#3899] [Q&A]
Hi,

i have a simple configuration that downloads using the `rss` plugin and some `regexp`. Sometimes, the downloaded files are RAR5, which can't be automatically read by my apps. In that case i need to manually convert the downloaded file to ZIP.

I see there is a [decompress](https://flexget.com/Plugins/decompress) plugin, but i can't see any conversion option, nor any `compress` plugin.

Is it possible to perform RAR5 to ZIP conversion using flexget ?

Thanks

**Comment by gazpachoking** [comment]
There isn't any built in plugin to do compression I don't think. But you may be able to do something with the [exec](https://flexget.com/en/Plugins/exec) plugin.
